benchexec/tablegenerator/util.py

```diff
@@ -192,14 +193,30 @@ def to_json(obj):
     return tempita.html(json.dumps(obj, sort_keys=True))
 
 
-def prettylist(list_):
+def prettylist(list_, merge=False):
+    """
+    Filter out duplicate values while keeping order.
+
+    In 'merge' mode we try to also match post-fixed names.
+    This might be useful in cases of several similar values like hostnames.
+    Example: ['test', 'pc1', 'pc2', 'pc3'] -> ['test', 'pc*']
```

Comment:
This explanation does not mention that this only collapses number suffixes and nothing else. This makes me wonder whether this is the right strategy.

Maybe we should just use `commonprefix` instead of removing the number? Maybe only if there is a certain number of different values with the same prefix?